sites/all/themes/orwell/templates/layouts/reol-site-template.tpl.php

```diff
@@ -47,15 +47,24 @@
   <?php if (!empty($content['footer'])): ?>
     <footer class="footer">
       <div class="footer__brand">
-
-          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" id="logo" class="branding__logo">
-            <img src="/<?php print path_to_theme(); ?>/assets/svg/eReolen-Red.svg" alt="<?php print t('Home'); ?>" />
-          </a>
-
+        <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" id="logo" class="branding__logo">
+          <img src="/<?php print path_to_theme(); ?>/assets/svg/eReolen-Red.svg" alt="<?php print t('Home'); ?>" />
+        </a>
       </div>
+
       <div class="footer__inner">
         <?php print render($content['footer']); ?>
       </div>
+
+      <div class="footer__ddf">
+        <a href="<?php print t('https://detdigitalefolkebibliotek.dk'); ?>" title="<?php print t('The Digital Public Library'); ?>" id="ddf-logo">
+          <img src="/<?php print path_to_theme(); ?>/assets/svg/DetDigitaleFolkebibliotek_logo_RGB_neg.svg" alt="<?php print t('The Digital Public Library'); ?>" />
+        </a>
+        <p>
+          <?php print t('eReolen is a part of The Digital Public Library'); ?><br/>
+          <a href="<?php print t('https://detdigitalefolkebibliotek.dk'); ?>"><?php print t('Read more at detdigitalefolkebibliotek.dk'); ?></a>
```

Comment:
Is making the actual url translatable (`t('https://…`) a clever trick to let an editor change it to something else using a translation or is it a mistake?